### DISCUSSION TITLE
FNIRSI GC-01: Installation successful, hacking

### DISCUSSION THREAD

**Discussion by godstargod** [Gissio/radpro#126] [General]
last month i got a gb01

just installed on my fnirsi gb01 with f103bt6 , that i changed the tube from the stock j6xx tube to a j321. i have nucleo boards but the procedure seemed to worked better wit ha stlink v2 clone i have.

i just want to ask in v2.0.2 i dont see any volume control

is it possible to have the device internal swdio and clock to be i2c ? or as the current state of the firmware can we poke at registers from the swd port ?

cheers thanks for this nice tool

**Comment by Gissio** [comment]
The SWD port requires an SWD signal.

Yes, you can access all registers through SWD.
